stdlib/public/core/IntegerParsing.swift

```diff
@@ -155,4 +155,27 @@ extension FixedWidthInteger {
     guard _fastPath(result != nil) else { return nil }
     self = result!
   }
+
+  /// Creates a new integer value from the given string.
+  ///
+  /// The string passed as `description` may begin with a plus or minus sign
+  /// character (`+` or `-`), followed by one or more numeric digits (`0-9`).
+  ///
+  ///     let x = Int("123")
+  ///     // x == 123
+  ///
+  /// If `description` is in an invalid format, or if the value it denotes in
+  /// base 10 is not representable, the result is `nil`. For example, the
+  /// following conversions result in `nil`:
+  ///
+  ///     Int(" 100")                       // Includes whitespace
+  ///     Int("21-50")                      // Invalid format
+  ///     Int("ff6600")                     // Characters out of bounds
+  ///
+  /// - Parameter description: The ASCII representation of a number in base 10.
+  @_semantics("optimize.sil.specialize.generic.partial.never")
+  @inline(__always)
```

Comment:
/cc @swiftix for the usage of the `@_semantics` attribute.
`@inline(__always)` is fine.